src/modules/musig/main_impl.h

```diff
@@ -45,64 +45,58 @@ static void secp256k1_musig_sha256_init_tagged(secp256k1_sha256 *sha) {
     sha->bytes = 64;
 }
 
-/* Compute r = SHA256(ell, idx). The four bytes of idx are serialized least significant byte first. */
-static void secp256k1_musig_coefficient(secp256k1_scalar *r, const unsigned char *ell, uint32_t idx) {
+/* Compute MuSig coefficient which is constant 1 for the second pubkey and
+ * SHA256(ell, x) otherwise. second_pk_x can be 0 in case there is no
+ * second_pk. Assumes both field elements x and second_pk_x are normalized. */
+static void secp256k1_musig_coefficient_internal(secp256k1_scalar *r, const unsigned char *ell, const secp256k1_fe *x, const secp256k1_fe *second_pk_x) {
```

Comment:
Is it possible to add VERIFY_CHECK here to verify that `x` and `second_pk_x` are normalized?
As pointed out at several place, I don't think this is the case currently.

Reply:
We already VERIFY_CHECK that _every_ low level function that expects a normalized field element gets a normalized field element (see  https://github.com/ElementsProject/secp256k1-zkp/blob/master/src/field_5x52.h#L17 and https://github.com/ElementsProject/secp256k1-zkp/blob/master/src/field_5x52_impl.h#L337)